dependencies.yaml

```diff
@@ -247,6 +281,9 @@ dependencies:
           # See https://github.com/zarr-developers/numcodecs/pull/475
           - numcodecs <0.12.0
           - packaging
+      - output_types: [pyproject]
+        packages:
+          - libkvikio==24.6.*
```

Comment:
libkvikio isn't a runtime requirement of kvikio, only a build-time requirement. Right now it's getting put into the wrong section in python/kvikio/pyproject.toml: it's going into project.dependencies, but you need it in build-system.requires.